releasenotes/notes/exit-if-sds-socket-not-found.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: security
+issue:
+- 45534
+releaseNotes:
+- |
+  **Added** prevent the istio-proxy from starting if the workload SDS socket is not found.
```

Comment:
Can we mention that

1. You need to set this variable to use external SDS now
2. what the variable is

Reply:
done